api/core/v1alpha1/clustercatalog_types.go

```diff
@@ -129,17 +129,20 @@ type ClusterCatalogStatus struct {
 	//  type: Image
 	// +optional
 	ResolvedSource *ResolvedCatalogSource `json:"resolvedSource,omitempty"`
-	// contentURL is a cluster-internal URL from which on-cluster components
-	// can read the content of a catalog
+	// catalogURLs contains the URLs that can be used to access the catalog.
 	// +optional
-	ContentURL string `json:"contentURL,omitempty"`
-
+	URLs *CatalogURLs `json:"catalogURLs,omitempty"`
```

Comment:
Nit: Use something like `urls`. It's already in the status of a `ClusterCatalog` and saying `catalogURLs` feels a bit redundant. Although not sure if it is a bit wonky to have an all lowercase "URL" field name.
```suggestion
	URLs *CatalogURLs `json:"urls,omitempty"`
```

Reply:
I had planned to try this, but honestly expected complaints from the linter or something else, since every other time we use the string 'url' as a typename/label, it universally insists on it being capitalized.  But it didn't this time.  
(Of course, I place very little confidence in it right now, since I'm passing all tests in my local but gh here is dying on everything ... so 🤷 )